aiohttp/test_utils.py

```diff
@@ -34,6 +34,67 @@ def unused_port():
         return s.getsockname()[1]
 
 
+class TestServer:
+    def __init__(self, app, *, protocol="http", host='127.0.0.1'):
+        self.app = app
+        self._loop = app.loop
+        self.port = unused_port()
```

Comment:
Grabbing `unused_port` in `__init__` may cause tricky bugs if `TestServer` is used explicitly (not through `test_server` fixture)